.eslintrc.yml

```diff
@@ -23,3 +23,4 @@ rules:
     no-unused-vars: off
     no-prototype-builtins: off
     linebreak-style: off
+    brace-style: 1tbs
```

Comment:
This change broke the CI "Lint" step.
It is missing a severity value.
Should be something like:
```
brace-style:
  - warn
  - 1tbs
```

Reply:
Fixed 👍 (I also removed ts-lint as it is deprecated and already part of the lint step)

Reply:
Looks fine.